appveyor.yml

```diff
@@ -1,5 +1,5 @@
 image:
-  - Visual Studio 2017
+  - Visual Studio 2019
```

Comment:
.NET Core 2.1 doesn't seem to be on the VS2017 image, so this change was necessary after removing the `<RuntimeFrameworkVersion>` element.